### PR TITLE
Fixed all mistakes in the Search page

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |

### CODE REVIEW

File: src/app/item-page/simple/field-components/preview-section/file-description/file-description.component.spec.ts [hunk 1]
```diff
@@ -58,7 +56,7 @@ describe('FileDescriptionComponent', () => {
     fileInput.id = 123;
     fileInput.name = 'testFile';
     fileInput.description = 'test description';
-    fileInput.fileSize = '5MB';
+    fileInput.fileSize = 2048;
```
Comment (MajoBerger) [inline src/app/item-page/simple/field-components/preview-section/file-description/file-description.component.spec.ts:59]: this is not 5MB like it was. Is it ok that this is 2MB?

File: src/assets/i18n/cs.json5 [hunk 2]
```diff
@@ -4013,8 +4015,8 @@
   "search.filters.applied.f.discoverable": "Soukromé",
   // "search.filters.applied.f.entityType": "Item Type",
   "search.filters.applied.f.entityType": "Typ položky",
-  // "search.filters.applied.f.has_content_in_original_bundle": "Has files",
-  "search.filters.applied.f.has_content_in_original_bundle": "Má soubory",
+  // "search.filters.applied.f.has_content_in_original_bundle": "Contain Files",
```
Comment (MajoBerger) [inline src/assets/i18n/cs.json5:4018]: This would be correct: `Contains Files`

File: src/assets/i18n/cs.json5 [hunk 3]
```diff
@@ -4107,8 +4109,8 @@
   "search.filters.filter.entityType.label": "Hledat typ položky",
   // "search.filters.filter.expand": "Expand filter",
   "search.filters.filter.expand": "Rozbalit filtr",
-  // "search.filters.filter.has_content_in_original_bundle.head": "Has files",
-  "search.filters.filter.has_content_in_original_bundle.head": "Má soubory",
+  // "search.filters.filter.has_content_in_original_bundle.head": "Contain Files",
```
Comment (MajoBerger) [inline src/assets/i18n/cs.json5:4112]: as above

File: src/assets/i18n/en.json5 [hunk 4]
```diff
@@ -4243,7 +4245,7 @@
 
   "search.filters.filter.expand": "Expand filter",
 
-  "search.filters.filter.has_content_in_original_bundle.head": "Has files",
+  "search.filters.filter.has_content_in_original_bundle.head": "Contain Files",
```
Comment (MajoBerger) [inline src/assets/i18n/en.json5:4248]: this should probably be Contains Files

File: src/assets/i18n/en.json5 [hunk 5]
```diff
@@ -4149,7 +4151,7 @@
 
   "search.filters.applied.f.entityType": "Item Type",
 
-  "search.filters.applied.f.has_content_in_original_bundle": "Has files",
+  "search.filters.applied.f.has_content_in_original_bundle": "Contain Files",
```
Comment (MajoBerger) [inline src/assets/i18n/en.json5:4154]: as below

**Review by MajoBerger — CHANGES_REQUESTED** [review]
some translations and comments are off

File: src/app/item-page/simple/field-components/preview-section/file-description/file-description.component.spec.ts (hunk 1 above)
Comment (milanmajchrak) [reply src/app/item-page/simple/field-components/preview-section/file-description/file-description.component.spec.ts:59]: It is OK, I just wanted to put there some value.

File: src/assets/i18n/cs.json5 (hunk 2 above)
Comment (milanmajchrak) [reply src/assets/i18n/cs.json5:4018]: Are you sure? I copied it from UFAL clarin DSpace: 
image

**Review by MajoBerger — APPROVED** [review]

**Pull request merged**